packages/@ember/application/tests/application_instance_test.js

```diff
@@ -21,9 +20,16 @@ moduleFor(
       super();
 
       document.getElementById('qunit-fixture').innerHTML = `
-      <div id='one'><div id='one-child'>HI</div></div><div id='two'>HI</div>
-    `;
-      application = run(() => Application.create({ rootElement: '#one', router: null }));
+        <div id='one'><div id='one-child'>HI</div></div><div id='two'>HI</div>
+      `;
+
+      application = run(() => {
+        let app;
+        ignoreDeprecation(() => {
```

Comment:
Is there anything about this test that is actually hitting the deprecation? Do we need to refactor the test harness to avoid using the globals resolver?

Reply:
We are not supplying a resolver, so it uses DefaultResolver by default.